includes/hub/admin/class-membership-plans-table.php

```diff
@@ -72,16 +51,42 @@ public function prepare_items() {
 	 * @return string
 	 */
 	public function column_default( $item, $column_name ) {
+		$memberships_list_url = sprintf( '%s/wp-admin/edit.php?s&post_status=wcm-active&post_type=wc_user_membership&post_parent=%d', $item['site_url'], $item['id'] );
+
 		if ( $column_name === 'network_pass_id' && $item[ $column_name ] ) {
 			return sprintf( '<code>%s</code>', $item[ $column_name ] );
 		}
-		if ( $column_name === 'links' ) {
-			$edit_url = get_edit_post_link( $item['id'] );
-			if ( isset( $item['node_url'] ) ) {
-				$edit_url = sprintf( '%s/wp-admin/post.php?post=%d&action=edit', $item['node_url'], $item['id'] );
+		if ( $column_name === 'network_pass_discrepancies' && isset( $item['network_pass_discrepancies'] ) && $item['network_pass_id'] ) {
```

Comment:
Should we be checking `$item['network_pass_id']` with `isset` as well? Will this always be present at this stage?

Reply:
No need for that, `network_pass_id` is not optional. `network_pass_discrepancies` is, because it's only included if the auditing features are enabled.